sdk/eventhub/Azure.Messaging.EventHubs.Processor/src/Azure.Messaging.EventHubs.Processor.csproj

```
@@ -14,7 +14,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Azure.Messaging.EventHubs" />
+    <PackageReference Include="Azure.Messaging.EventHubs"/>
```

```suggestion
    <PackageReference Include="Azure.Messaging.EventHubs" />
```

this was unintentional and will be reverted after the test runs complete.